gm4_villager_telltale/data/gm4_villager_telltale/functions/main.mcfunction

```diff
@@ -0,0 +1,7 @@
+
+schedule function gm4_villager_telltale:main 3s
+
+execute as @e[type=villager,tag=gm4_vt_looking] run function gm4_villager_telltale:villager/stop_looking
+
+tag @a remove gm4_tv_holding_villager_block
```

Comment:
This tag should be removed at the end of the `player` function. There's no reason to run it every clock cycle.

Reply:
Also why is it `tv` instead of `vt`?